egs2/libritts/speechlm1/run_acadamic.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
Let's sync to one run.sh for the start.

Reply:
Sure. Some of the files are not for review/public but are just used during development. Will clean up the files after we finish it. Sorry for this inconvenience.